GeoIP2.UnitTests/project.json

```diff
@@ -11,7 +11,7 @@
   },
 
   "commands": {
-    "test": "MaxMind.Db.Test"
+    "test": "MaxMind.GeoIP1.UnitTests"
```

Comment:
Hehe, I wonder where that 1 came from..  ¯\ (°_o)/¯ 
Fixed in c0754db. Thanks! :)